### DISCUSSION TITLE
NER portuguese

### DISCUSSION THREAD

**Discussion by kennynakamura** [sepinf-inc/IPED#595] [Q&A]
The NER implementation support Portuguese? 
I saw that CoreNLP it does not have a specific portuguese model.

**Comment by lfcnassif** [comment]
Users can choose between 3 implementations: StanfordCoreNLP, Apache OpenNLP, Python NLTK. CoreNLP doesn't have a portuguese model, not sure about others.

**Reply by kennynakamura** [reply]
I know that NLTK have. I made an implementation with Portuguese Spacy (NLP model), but i guess NTLK is more easy to configure.
